openfisca_country_template/tests/reforms/modify_social_security_brackets.yaml

```diff
@@ -0,0 +1,31 @@
+# Test files describe situations and their expected outcomes
+# We can run this test on our command line using `openfisca-run-test social_security_contribution.yaml`
+
+# This test is a partial adaptation of `social_security_contribution.yaml` to match the modifications introduced
+# by the reform of the same name.
```

Comment:
I think we rely on line wrapping on other comments. If that's the case, please be consistent.